tests/test_process.py

```diff
@@ -129,50 +129,30 @@ def test_get_lc2_path():
 
 @responses.activate
 def test_get_s2_metadata_not_found():
-    responses.add(responses.GET, f'{process.S2_SEARCH_URL}/foo', status=404)
-    responses.add(
-        responses.POST, process.S2_SEARCH_URL,
-        body='{"numberReturned": 0}', status=200,
-    )
+    responses.add(responses.GET, f'{process.S2_GRANULE_DIR}////foo.SAFE/manifest.safe', status=404)
     with pytest.raises(ValueError):
         process.get_s2_metadata('foo')
 
 
 @responses.activate
-def test_get_s2_metadata_cog_id():
-    responses.add(
-        responses.GET, f'{process.S2_SEARCH_URL}/2FS2B_22WEB_20200913_0_L2A',
-        body='{"foo": "bar"}', status=200,
-    )
-
-    assert process.get_s2_metadata('2FS2B_22WEB_20200913_0_L2A') == {'foo': 'bar'}
+def test_get_s2_metadata():
+    # TODO implement me
+    pass
 
 
-@responses.activate
-def test_get_s2_metadata_esa_id():
-    responses.add(
-        responses.GET, f'{process.S2_SEARCH_URL}/S2B_MSIL2A_20200913T151809_N0214_R068_T22WEB_20200913T180530',
-        status=404,
-    )
-    responses.add(
-        responses.POST, process.S2_SEARCH_URL,
-        body='{"numberReturned": 1, "features": [{"foo": "bar"}]}', status=200,
-    )
+def test_get_s2_path(test_data_directory):
+    scene_name = 'S2A_MSIL1C_20160616T112217_N0204_R137_T29QKF_20160617T193500'
+    f = open(f'{test_data_directory}/{scene_name}.manifest.safe', 'r')
+    manifest_text = f.read()
+    path = process.get_s2_path(manifest_text, scene_name)
+    assert path == '/vsicurl/https://storage.googleapis.com/gcp-public-data-sentinel-2/tiles//./GRANULE' \
+                   '/S2A_OPER_MSI_L1C_TL_SGS__20160616T181414_A005139_T29QKF_N02.04/IMG_DATA' \
+                   '/S2A_OPER_MSI_L1C_TL_SGS__20160616T181414_A005139_T29QKF_B08.jp2'
 
-    assert process.get_s2_metadata('S2B_MSIL2A_20200913T151809_N0214_R068_T22WEB_20200913T180530') == {"foo": "bar"}
 
-
-@responses.activate
-def test_get_s2_metadata_json():
-    responses.add(responses.GET, f'{process.S2_SEARCH_URL}/S2B_60CWT_20220130_0_L1C', status=404)
-
-    responses.add(
-        responses.POST, process.S2_SEARCH_URL,
-        body='{"numberReturned": 0}', status=200,
-    )
-
-    s3_path = process.get_s2_metadata('S2B_60CWT_20220130_0_L1C')['assets']['B08']['href']
-    assert s3_path == 's3://sentinel-s2-l1c/tiles/60/C/WT/2022/1/30/0/B08.jp2'
+def test_get_s2_bbox(test_data_directory):
+    bbox = process.get_s2_bbox(str(test_data_directory / 'T60CWU_20160414T200612_B08.jp2'))
+    assert bbox == [-183.0008956, -78.4606571, -178.0958227, -77.438842]
```

Comment:
This covers the antimeridian example; we can copy/paste a similar test for `T55XEE_20200911T034541_B08.jp2` to test a non-antimeridian case.